src/javascripts/modules/ExpandList.js

```diff
@@ -0,0 +1,32 @@
+import debounce from 'lodash.debounce'
+
+export default class ExpandList {
+  constructor(el) {
+    this.el = el
+    this.tagArray = Array.from(this.el.querySelectorAll('.tag-list__item'))
+    this.btnArray = Array.from(this.el.querySelectorAll('.tag-list__btn'))
+
+    this.bindEvents()
+  }
+
+  bindEvents() {
+    this.btnArray.forEach(btn => btn.addEventListener('click', this.expand))
+    window.addEventListener('resize', debounce(this.resize, 250))
+  }
+
+  //for expanding tag-list__items when clicking tag-list__btns
+  expand = (e) => {
```

Comment:
Not worth changing, but for future reference, when passing a _single_ argument like this, you don't need the parens, but you do for multiple, e.g.:
```js
expand = e => { ... }
doSomethingElse = (arg1, arg2, arg3) => { ... }
```